Component/LinearSolver/Direct/src/sofa/component/linearsolver/direct/SparseCholeskySolver.inl

```diff
@@ -30,7 +31,12 @@ template<class TMatrix, class TVector>
 SparseCholeskySolver<TMatrix,TVector>::SparseCholeskySolver()
     : f_verbose( initData(&f_verbose,false,"verbose","Dump system state at each iteration") )
     , S(nullptr), N(nullptr)
+    , type_perm(initData(&type_perm, "permutation", "Type of fill reducing permutation"))
 {
+    computePermutation = true;
```

Comment:
Put the initialization of `type_perm` into the constructor initializer list (https://en.cppreference.com/w/cpp/language/constructor).